docs/contributing/feature-request-process.md

```diff
@@ -0,0 +1,32 @@
+# Request a Feature
+
+If you have a great idea for an improvement of the platform, or any other feedback, please make sure to open a new
+Discussion on our GitHub Discussions board: https://github.com/directus/directus/discussions
+
+## Opening a New Feature Request
+
+New feature requests can be opened under the Draft Requests section of
+[GitHub Discussions](https://github.com/directus/directus/discussions). Feature Requests follow a Request-for-Comments
+format that allows anybody to fully understand what you're proposing, and help speed up the review and triaging process.
+
+**While the form might seem intimidating at first, please do fill out all sections in as much detail as you can. The
+less ambiguity exists around _how_ a feature should work, the easier it is to review, triage, and subsequently develop a
+feature.**
+
+## Review Process
+
+Once a feature request gets 15 or more upvotes, it will be moved to the "Under Review" status at which point the
+Directus Core team will review it to decide if it will be approved, denied, or if changes are required. Once the feature
+request is accepted, it will be moved to the "Approved Requests" status, at which point the feature is ready to be
+implemented.
+
+However, if a feature request didn't reach the required 15 upvotes after 3 months of time, it will be closed due to a
+lack of community interest.
```

Comment:
```suggestion
However, if a feature request doesn't reach the at least 15 upvotes 3 months after it is created, it will be closed due to a lack of community interest.
```